Source/OCMock/OCMMacroState.m

```diff
@@ -45,7 +45,10 @@ + (OCMStubRecorder *)endStubMacro
 	if([recorder wasUsed] == NO)
 	{
 		[NSException raise:NSInternalInconsistencyException
-					format:@"Mock object was not used in OCMStub/OCMExpect/OCMReject. Did you accidentally use a real object?"];
+					format:@"Did not record an invocation in OCMStub/OCMExpect/OCMReject.\n"
+						   @"Possible causes are:\n"
+						   @"- The target is not a mock object.\n"
```

Comment:
We should refer to a *receiver* and not *target*. That's what Apple uses to refer to an object on which a method is invoked.

Reply:
Fixed.